windows_route53.tf

```diff
@@ -0,0 +1,11 @@
+# Private DNS A record for Windows instance
```

Comment:
Minor, but slightly more accurate:
```suggestion
# Private DNS A record for Windows instances
```

Reply:
Oops, I copied that from `kali_route53.tf` and didn't notice.